kartothek/core/dataset.py

```diff
@@ -907,6 +907,37 @@ def from_dataset(dataset):
         ds_builder.partitions = dataset.partitions
         return ds_builder
 
+    def modify_uuid(self, tgt_uuid: str):
+        """
+        Modify the dataset uuid and depending metadata:
+        - paths to partitioning files
+        - path to index files
+
+        :param tgt_uuid:
```

Comment:
The rest of the code bases uses numpy style docstrings. Can you change them to be consistent?

Reply:
OK, changed the comment style. (Not sure if the doctrings I've seen so far were consistent throughout the whole project, btw...)

Reply:
They are not consistent unfortunately, but at least the intention is to stick with numpy...